SocialCareCaseViewerApi/V1/Gateways/ICaseStatusGateway.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+using SocialCareCaseViewerApi.V1.Domain;
+
+namespace SocialCareCaseViewerApi.V1.Gateways
+{
+    public interface ICaseStatusGateway
+    {
+        public List<CaseStatus> GetCaseStatusesByPersonId(long personId);
```

Comment:
We should be returning an IEnumerable<CaseStatus>, no need to enforece the structure of the data returned by the gateway when possible